src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/virtual_appliance/_create.py

```diff
@@ -136,6 +111,14 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Properties",
             help="Network Profile containing configurations for Public and Private NIC.",
         )
+        _args_schema.nva_interface_configurations = AAZListArg(
+            options=["--interface-config", "--nva-interface-configurations"],
```

Comment:
alias and the original option name need to maintain the same singular or plural form.